### PR TITLE
ceph-*-build/build/build_rpm: avoid "No such file or directory" errors

### PR DESCRIPTION
**Pull request by tchaikov**

### CODE REVIEW

**Comment by djgalloway** [comment]
How does this fix "No such file or directory" errors?  Does it fix the ceph-container error specifically?

e.g.,

```
[91merror: open of null/noarch/ceph-release-1-1.el8.noarch.rpm failed: No such file or directory
```

I filed https://github.com/ceph/ceph-container/pull/1709 hoping to fix that specific one but even bumping the version to `1-1` still failed.  https://jenkins.ceph.com/job/ceph-dev-new-build/ARCH=x86_64,AVAILABLE_ARCH=x86_64,AVAILABLE_DIST=centos7,DIST=centos7,MACHINE_SIZE=gigantic/42992/console

**Comment by tchaikov** [comment]
@djgalloway the goal of this change is two folded

1. to silence the warnings like

```
cp: cannot stat 'rpm/*.patch': No such file or directory
```
see

https://jenkins.ceph.com/job/ceph-dev-new-build/ARCH=x86_64,AVAILABLE_ARCH=x86_64,AVAILABLE_DIST=centos8,DIST=centos8,MACHINE_SIZE=gigantic/43088//consoleFull

2. and to consolidate the code base piece meal.

**Comment by djgalloway** [comment]
Yeah, I was more focused on the ceph-release RPM missing `No such file` errors.  Those are extremely disruptive right now.

File: scripts/build_utils.sh [hunk 1]
```diff
@@ -1081,3 +1081,30 @@ maybe_reset_ci_container() {
         CI_CONTAINER=false
     fi
 }
+
+setup_build_area() {
```
Comment (djgalloway) [inline scripts/build_utils.sh:1085]: Can you rename this to `setup_rpm_build_area()` ?

**Comment by tchaikov** [comment]
> Yeah, I was more focused on the ceph-release RPM missing `No such file` errors. Those are extremely disruptive right now.

i knew. i wanted to fix it as well. just need more time.

File: scripts/build_utils.sh (hunk 1 above)
Comment (tchaikov) [reply scripts/build_utils.sh]: renamed.

**Pull request merged**